catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/SubscriptionsPersistentStoreImpl.java

```diff
@@ -237,6 +237,20 @@ private String quote(String value) {
     return Collections.emptySet();
   }
 
+  @Override
+  public void removeSubscriptions(String id) {
+    final String ecql = queryId(id);
+
+    LOCK.lock();
+    try {
+      persistentStore.delete(SubscriptionsPersistentStore.SUBSCRIPTIONS_TYPE, ecql, 0, 1000);
+    } catch (PersistenceException e) {
+      LOGGER.warn("Could not delete subscriptions for query {}", ecql, e);
```

Comment:
✏️ stack traces should be logged at debug level and not at warning level.

Reply:
Fixed.